### PR TITLE
Don't lowercase linguistics annotations

### PR DESCRIPTION
**Pull request by bratseth**
Tokens are already lowercased by our bundled linguistics components. Lowercasing again when annotating precludes plugging in a lingustics component which preserves casing.

### CODE REVIEW

**Comment by bratseth** [comment]
@arnej27959 AND @baldersheim please review

**Review by baldersheim — DISMISSED** [review]

File: indexinglanguage/src/test/java/com/yahoo/vespa/indexinglanguage/linguistics/LinguisticsAnnotatorTestCase.java [hunk 1]
```diff
@@ -90,7 +90,7 @@ public void requireThatTermAnnotationsAreLowerCased() {
                 if (!specialToken && !type.isIndexable()) {
                     continue;
                 }
-                assertAnnotations(expected, "foo", newToken("foo", "BAR", type, specialToken));
+                assertAnnotations(expected, "foo", newToken("foo", "bar", type, specialToken));
```
Comment (arnej27959) [inline indexinglanguage/src/test/java/com/yahoo/vespa/indexinglanguage/linguistics/LinguisticsAnnotatorTestCase.java:93]: please change this test to "requireThatTermAnnotationsAreNotLowerCased", keep "BAR" as input and check that the annotation contains BAR

**Review by arnej27959 — CHANGES_REQUESTED** [review]
unit test should be updated

**Review by arnej27959 — APPROVED** [review]
👍

**Comment by arnej27959** [comment]
this broke a system test; "Chicago" was not lowercased when indexing.  So bundled linguistics did not lowercase it after all?

**Comment by bratseth** [comment]
Yes, looking into it ...

**Pull request merged**